R/fit_process.R

```diff
@@ -987,9 +987,20 @@ extract_severity <- function(samples) {
 
 extract_severity_region <- function(state, step, date) {
 
-  what <- c("ifr", "ihr", "hfr",
-            "ifr_strain_1", "ihr_strain_1", "hfr_strain_1",
-            "ifr_strain_2", "ihr_strain_2", "hfr_strain_2")
+  aggregated <- c("ifr", "ihr", "hfr")
+  disag <- c("ifr_disag_", "ihr_disag_", "hfr_disag_")
+  suffix <- paste0("_age_", c(sircovid:::sircovid_age_bins()$start))
+  vacc_class <- c("", "_1", "_2", "_3", "_4")
```

Comment:
Please code this more flexibly - I imagine we potentially want this to work for a flexible number of vaccine classes

Reply:
This is now addressed in commit https://github.com/mrc-ide/spimalot/pull/214/commits/d21ee110548605cffc950e3795f999c305a4eb60